controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -14,7 +14,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	flowsv1alpha1 "github.com/netobserv/network-observability-operator/api/v1alpha1"
+	flowsv1beta1 "github.com/netobserv/network-observability-operator/api/v1beta1"
```

Comment:
we could perhaps use an import alias like `flowslatest` to avoid having to rename everywhere in the code.. on version upgrades, we would just need to update the imports. That would also be less conflicts during rebase/merges.